apple/Sources/Sargon/Extensions/Methods/RET/TransactionManifest+Wrap+Functions.swift

```diff
@@ -39,10 +39,10 @@ extension TransactionManifest {
         transactionManifestSummary(manifest: self)
     }
     
-    public func executionSummary(encodedReceipt: Data) throws -> ExecutionSummary {
+    public func executionSummary(engineToolkitReceipt: Data) throws -> ExecutionSummary {
         try transactionManifestExecutionSummary(
```

Comment:
we should document how Hosts retrive and provide this data, here and in Rust fn.

Reply:
will add!